src/components/AllJobLogsView/AllJobLogsView.js

```diff
@@ -160,15 +163,7 @@ export const AllJobLogsViewComponent = ({
       />
     ));
 
-  const checkObjectValues = obj => {
-    const values = Object.values(obj);
-
-    if (Object.keys(obj).length === 0 || obj.sort === '-completedDate' || obj.qindex === 'hrId') {
-      return true;
-    }
-
-    return !values.every(value => value !== undefined);
-  };
+  const isResetButtonDisabled = location.search === initialQuery || location.search === defaultSortingSearch;
```

Comment:
location.search can be moved to separate variable or just to `const {search} = location`, because used 3 times

Reply:
@vashjs done